src/stores/ScannerStore.js

```diff
@@ -17,10 +17,9 @@
 // @flow
 import { Container } from 'unstated';
 import transaction from '../util/transaction';
-import { keccak, ethSign, brainWalletSign, decryptData } from '../util/native';
-import { saveTx, loadAccountTxs } from '../util/db';
-import { accountId } from '../util/account';``
-import { NETWORK_TITLES, NETWORK_IDS } from '../constants';
+import { keccak, brainWalletSign, decryptData } from '../util/native';
```

Comment:
The rule is: one feature (logical change) - one PR. Make a separate PR for these changes, please.

Reply:
:+1: